apps/docs/content/guides/local-development/customizing-email-templates.mdx

```diff
@@ -104,7 +103,7 @@ There are several Auth email templates which can be configured. Each template se
 
 ## Available notification email templates
 
-There are several Auth notification email templates which can be configured. These are security-related notifications sent to users, only if they have been enabled:
+There are several security-related email templates which can be configured. These notifications are only sent to users if they have been enabled:
 
 ### `auth.email.notification.password_changed`
```

Comment:
@issuedat we refer to these types of emails as “Notifications” in some places, “Security” in others. Do you think this is a possible point of future user confusion? Should we consolidate?

<img width="800" height="576" alt="CleanShot 2025-11-06 at 11 08 39@2x" src="https://github.com/user-attachments/assets/b0529d39-b0f1-4313-90a9-2237bd7260e3" />